.github/pr_labeler.yml

```diff
@@ -50,8 +54,6 @@ tailscale:
   - plugins/source/tailscale/**/*
 vercel:
   - plugins/source/vercel/**/*
-salesforce:
-  - plugins/source/salesforce/**/*
 azblob:
   - plugins/destination/azblob/**/*
 postgresql:
```

Comment:
No good deed goes unpunished: because of the sorting this file now has some conflicts 😅

Reply:
yeah I know, I'll keep on top of it until we get it merged